### ISSUE TITLE
[Feature request] option for monochrome output

### ISSUE THREAD

**Issue by soonic6** [tpill90/steam-lancache-prefill#128]
The color generates cryptic outputs in my logfiles, maybe an option for a monochrome output will be possible?

![image](https://user-images.githubusercontent.com/73687921/189481793-1ea183e6-f596-4511-91f5-ed179ba626d8.png)

see also: https://github.com/tpill90/battlenet-lancache-prefill/issues/54

**Comment by tpill90** [comment]
Commands used to reproduce this: 
```
#!/bin/bash
 
#Variablen
PI_IP="debianautofill.fritz.box"
SSH_USER="MY_USER"
SSH_PW="MY_PASSWORD"
CONTAINERNAME="debian_autofill"
USERNAME="MY_USER"
 
lxc-start --name $CONTAINERNAME
sleep 10
 
 
lxc-attach --name $CONTAINERNAME -- su $USERNAME bash -c "env TERM=xterm /home/sonic6/steam_prefill_update.sh"
lxc-attach --name $CONTAINERNAME -- su $USERNAME bash -c "env TERM=xterm /home/sonic6/steam_prefill/SteamPrefill prefill"
lxc-attach --name $CONTAINERNAME -- su $USERNAME bash -c "env TERM=xterm /home/sonic6/battlenet_prefill/BattleNetPrefill prefill --products odin"
 
lxc-stop --name $CONTAINERNAME
```

**Comment by tpill90** [comment]
![image](https://user-images.githubusercontent.com/766513/194437301-82c03c61-bec6-4560-9bc7-6f99c2d1096c.png)

**Comment by tpill90** [comment]
Released in [v1.6.0](https://github.com/tpill90/steam-lancache-prefill/releases/tag/v1.6.0)
